torch/_dynamo/debug_utils.py

```diff
@@ -151,14 +151,14 @@ def can_convert_to_string(gm):
         return True
 
     @staticmethod
-    def convert(gm):
+    def convert(gm, imports=True):
         from torch.nn.modules.module import _addindent
 
         tab = " " * 4
 
         model_str = textwrap.dedent(
-            """
-            from torch.nn import *
+            f"""
+            {'from torch.nn import *' if imports else ''}
```

Comment:
This seems a bit questionable

Reply:
If you keep this here, you cannot nest this class. You can only have * imports top level.